src/NuGet.Core/NuGet.Versioning/EmptyVersionRange.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Versioning
+{
+    /// <summary>
+    /// Represents a <see cref="VersionRange"/> that has empty normalized version.
+    /// </summary>
+    public class EmptyVersionRange : VersionRange
```

Comment:
Do we still need this?

Reply:
No - removed.

Reply:
Missed to add the file delete in the commit. Added.